vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/events/CustomValueSetEvent.java

```diff
@@ -0,0 +1,30 @@
+package com.vaadin.flow.component.combobox.events;
```

Comment:
Let's add a license header comment to this new file.

Reply:
Thanks, done